dm_control/locomotion/walkers/assets/dog_v2/dog_base.xml

```diff
@@ -157,14 +158,24 @@
     <default class="sensor">
       <site group="4"/>
     </default>
+    <default class="wrapping">
+      <geom type='cylinder' mass="0" contype="0" conaffinity="0" group="2" rgba="0 0 1 1" euler="90 0 0"/>
+      <site group="2" rgba="0 0 1 1" />
+    </default>
+    <default class="marker">
+      <site group="3" rgba="1 0.5 0 0.5" size="0.01 0.01 0.01"/>
+    </default>
+    <default class="mocap_marker">
+      <site group="3" rgba="0 1 0 0.5" size="0.01 0.01 0.01"/>
+    </default>
   </default>
 
   <asset>
     <texture name="grid" type="2d" builtin="checker" rgb1="0.1 0.2 0.3" rgb2="0.2 0.3 0.4" mark="edge" markrgb="0.2 0.3 0.4" width="300" height="300"/>
+    <texture type="skybox" name="skybox" builtin="gradient" mark="random" rgb1="0.4 0.6 0.8" rgb2="0 0 0" markrgb="1 1 1" width="800" height="800"/>
```

Comment:
See my earlier comment. If possible, remove the skybox and grid materials from the XML, or make width and height much smaller because they take up unnecessary RAM.